packages/react/src/components/header/components/navigationLink/NavigationLink.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { axe } from 'jest-axe';
+
+import { NavigationLink } from './NavigationLink';
+import { HeaderNavigationMenuWrapper } from '../../../../utils/test-utils';
+
+describe('<NavigationLink /> spec', () => {
+  it('renders the component', () => {
+    const { asFragment } = render(<NavigationLink href="#" label="Link" />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it('should not have basic accessibility issues', async () => {
+    const { container } = render(<NavigationLink href="#" label="Link" />);
```

Comment:
There could be a linebreak between test cases.
I'll fix the template at some point 😁